definitions/features/foreman_tasks.rb

```diff
@@ -82,6 +82,10 @@ def condition(state)
     end
   end
 
+  def resume_task_using_hammer
+    hammer('task resume')
```

Comment:
@iNecas
Hammer module is already included in `lib/foreman_maintain/feature.rb`. That's why we don't need to include hammer module here.